:100755 100755 b5f2539 0000000 M	razer_battery_tray.py

diff --git a/razer_battery_tray.py b/razer_battery_tray.py
index b5f2539..b4c8b12 100755
--- a/razer_battery_tray.py
+++ b/razer_battery_tray.py
@@ -16,7 +16,7 @@ UPDATE_INTERVAL_IN_SECS = 1 / 5
 print("Starting Razer Battery Tray")
 
 script_path = os.path.dirname(__file__)
-icons_path = os.path.join(script_path, "icons")
+icons_path = os.path.join(script_path, '..', 'share', 'razerBatteryTray', 'icons')
 
 if hasattr(PIL.Image, "Resampling"):  # Pillow >= 9.1.0
     DOWNSCALE_METHOD = PIL.Image.Resampling.LANCZOS
